### PR TITLE
completed-project

### PR DESCRIPTION
**Pull request by coder-mezab**
i have completed the card hover effect project let me know what your thoughts are!!!

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  min-height: 100vh;
+  background: #333;
+  justify-content: center;
+  align-items: center;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
```
Comment (faran4engg) [inline css/style.css:9]: why should body care about these prop ?

```css
  display: flex;
  justify-content: center;
  align-items: center;
  
  min-height: 100vh;

```

Comment (faran4engg) [reply css/style.css:9]: where's box-sizing ?

Comment (coder-mezab) [reply css/style.css:9]: i will solve it
@faran4engg

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  display: flex;
+  margin: auto;
+  position: relative;
+  background: white;
+  justify-content: center;
+  align-items: center;
+  box-shadow: 0 0 30px black;
+  border-radius: 10px 10px 0 0;
+  transform: translate(0, 100px);
+  z-index: 1;
```
Comment (faran4engg) [inline css/style.css:39]: again.... group the related properties @coder-mezab

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  position: relative;
+  margin: auto;
+  display: flex;
+  padding: 20px;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  border-radius: 0 0 10px 10px;
+  box-shadow: 0 0 30px black;
+  transform: translateY(-100px);
```
Comment (faran4engg) [inline css/style.css:71]: same here.. group them

**Review by faran4engg — COMMENTED** [review]
Overall looks super nice... great job... just fix minor comments... once approved, merge it

**Comment by coder-mezab** [comment]
ok @faran4engg

**Review by faran4engg — APPROVED** [review]
LGTM 👍

**Pull request merged**